packages/backend/src/workers/flow.ts

```diff
@@ -62,7 +62,7 @@ worker.on('completed', (job) => {
 })
 
 worker.on('failed', (job, err) => {
-  logger.info(
+  logger.error(
```

Comment:
niceee, but could we differentiate the worker error messages among action, flow and trigger since I cannot find an easy way to detect the difference on datadog?